prow/config/config_test.go

```diff
@@ -294,3 +297,17 @@ func TestBazelJobHasContainerPort(t *testing.T) {
 		t.Errorf("Error in periodic: %v", err)
 	}
 }
+
+func TestSortedTests(t *testing.T) {
+	config, err := ioutil.ReadFile("../config.yaml")
+	if err != nil {
+		t.Fatalf("Could not load config: %v", err)
+	}
+	expected, err := exec.Command("../../jobs/config_sort.py", "--dump-sorted-prow-config").CombinedOutput()
```

Comment:
maybe import `jobs/config_sort` and call a function rather than shelling out, since this is all python?